charts/vault-operator/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 type: application
 name: vault-operator
-version: 1.20.0-dev.1
-appVersion: 1.19.0
+version: 1.20.1
+appVersion: 1.20.0
```

Comment:
No need to increase versions manually anymore. Please rebase your PR.

Reply:
Hey @sagikazarmark, thanks for your valuable feedback. I am no longer updating the versions + I have rebased the branch.